packages/sync/src/Replicastore.php

```diff
@@ -1288,10 +1288,15 @@ public function checksum_histogram( $object_type, $buckets, $start_id = null, $e
 			$where .= " AND $id_field <= " . intval( $end_id );
 		}
 
+		$distinct = '';
+		if ( 'term_relationships' === $object_type ) {
+			$distinct = 'DISTINCT';
```

Comment:
Nit: should one of the spaces be here, so the query doesn't end up with two spaces if we're currently dealing with non-term-relationships?

Reply:
the double space doesn't bother me.